client/deviceadm/client_deviceadm.go

```diff
@@ -43,19 +44,20 @@ type ClientConfig struct {
 
 // ClientRunner is an interface of device admission client
 type ClientRunner interface {
-	AddDevice(req AdmReq, client requestid.ApiRequester) error
-	log.ContextLogger
+	AddDevice(ctx context.Context, req AdmReq, client requestid.ApiRequester) error
 }
 
 // Client is an opaque implementation of device admission client. Implements
 // ClientRunner interface
 type Client struct {
-	log  *log.Logger
 	conf ClientConfig
```

Comment:
Can you also `gorename -from "github.com/mendersotware/deviceauth/client/deviceadm".ClientConfig -to Config` in a separate patch?